backtesting/test/__init__.py

```diff
@@ -15,6 +15,8 @@ def _read_file(filename):
 EURUSD = _read_file('EURUSD.csv')
 """DataFrame of hourly EUR/USD forex data from April 2017 to February 2018."""
 
+AMZN = _read_file('AMZN.csv')
+"""DataFrame of daily NASDAQ:AMZN (Amazon) stock price data from 15/05/1997 to 23/04/2021."""
```

Comment:
I'm quite opposed to vendoring a new data set in a bug fix like this. I meant if you're able to easily construct a small, contrived unit test / test case, i.e., something that fails on `atr[-1]`, but not with `atr[index]`. Otherwise, no test is preferred to this. :grimacing:

Reply:
Yea. It's little difficult to test atr[-1] unless I write that piece of code out in the test case to do the computation But then it won't really be considered as a unit test for the TrailingStrategy. Rather it would be a logic test. Is it okay to skip the test for this one then? I can add descriptive comment in that line of code as to why I am using index instead of -1.